src/tests/unit/tests/DetailsView/components/issue-filing-dialog.test.tsx

```diff
@@ -66,23 +66,18 @@ describe('IssueFilingDialog', () => {
             selectedBugFilingServiceData: selectedServiceData,
             bugFileTelemetryCallback: telemetryCallbackMock.object,
         };
-
-        isSettingsValidMock.setup(isSettingsValid => isSettingsValid(selectedServiceData)).verifiable();
-        createBugFilingUrlMock
-            .setup(createBugFilingUrl => createBugFilingUrl(selectedServiceData, selectedBugDataStub, envInfo))
-            .verifiable();
     });
 
-    it('render open', () => {
-        const testSubject = shallow(<IssueFilingDialog {...props} />);
+    it.each([true, false])('isSettingsValid: %s', isSettingsValid => {
+        isSettingsValidMock
+            .setup(isValid => isValid(selectedServiceData))
+            .returns(() => isSettingsValid)
+            .verifiable();
```

Comment:
I don't think you need to verify this call as you are returning a boolean that definitely change what's in the snapshot

Reply:
Resolved without change. Can you clarify?

Reply:
yes, it makes 'isHidden' different in snapshot.